frontend_tests/node_tests/templates.js

```diff
@@ -519,13 +519,13 @@ run_test('compose_invite_users', () => {
 run_test('compose_all_everyone', () => {
     const args = {
         count: '101',
-        name: 'all',
+        mention: 'all',
     };
     const html = render('compose_all_everyone', args);
     const button = $(html).find("button").first();
     assert.equal(button.text(), "translated: Yes, send");
     const error_msg = $(html).find('span.compose-all-everyone-msg').text().trim();
-    assert.equal(error_msg, "translated: Are you sure you want to mention all 101 people in this stream?");
+    assert.equal(error_msg, "translated: Are you sure you want to mention all 101 people in this stream?  This will send email and mobile push notifications to most of those 101 users.  If you don't want to do that, please edit your message to remove the @all mention.");
```

Comment:
We should probably add test cases for each wildcard since we're dealing with them separately now.

Reply:
Sure. I'll add separate tests for @all, @everyone and @stream.